ONSdigital/census-rm-exception-manager-2021
# Enhancements to allow replaying messages and other stuff

# Motivation and Context
We're slowly improving the capabilities of our operational tooling. This wave of enhancement adds the ability to do various things, including replaying quarantined messages.

# What has changed
Added some new endpoints the the API.

# How to test?
Try quarantining a message, then replay it by hitting the `/replayquarantinedmessage/<id>` endpoint... the message should pop back onto the Rabbit queue it came from (but beware - it might get skipped again unless you reset the exception manager).

You can also try out the endpoints to get all the auto quarantine rules, or delete an auto quarantine rule, if that's your sort of thing.

# Links
Trello: https://trello.com/c/whSpMIJG
